testsuite/fuzzer/Cargo.toml

```diff
@@ -13,7 +13,7 @@ bcs = { workspace = true }
 clap = "4.5.20"
```

Comment:
Also could be made `{ workspace = true }`?

Reply:
Done